R/calc_birdNBuse.R

```diff
@@ -16,25 +17,29 @@
 #' The function assumes that NestBoxID = LocationID.
 #'
 #' @param Brood_data Data frame. Data frame with standardized brood data.
-#' @return A data frame with the NestBoxID and corresponding FirstSeason and LastSeason.
+#' @return A data frame with all combinations of `PopID` and `NestBoxID`
+#'  and corresponding `StartSeason` and `EndSeason`.
 #' @export
 #'
 #' @examples
 #' library(dplyr)
-#' Brood_data <- tibble::tibble(BroodID = c(1:10), LocationID = LETTERS[c(1,3,1,1,2,3,1,2,1,2)],
-#'                              BreedingSeason = sort(sample(2012:2016, 10, replace = TRUE)))
-#' calc_birdNBuse(Brood_data)
+#' Brood_data <- tibble::tibble(BroodID = rep(c(1:5), 2),
+#'                              PopID = rep(c('XXX', 'YYY'), each = 5),
+#'                              LocationID = LETTERS[c(1,2,1,1,2,2,1,2,1,2)],
+#'                              BreedingSeason = c(2001, 2001, 2002, 2003, 2003,
+#'                              1962, 1963, 1963, 1965, 1965))
+#'calc_birdNBuse(Brood_data)
```

Comment:
Could you include something like this as a small test for `calc_birdNBuse` (and also include a test for the ReRinging code too).

Reply:
I've made a small test for `calc_birdNBuse`. Test for ReRinging code we should make after fixing the function.